README.md

```diff
@@ -73,7 +73,62 @@ The Ansible community website is built with [Nikola](https://getnikola.com/).
 
 > Nikola configuration for the website is in ``conf.py``.
 
-## Checking your changes
+## Site accessibility
+
+Being considerate of users with assistive technology is a central part of the Ansible community pledge to be inclusive.
+When adding a blog post or making changes to the templates, we ask that you perform some accessibility checks.
+This ensures that everyone can read your post and our site remains open and accessible to all.
+
+### Web Content Accessibility Guidelines (WCAG) 2.1
+
+Changes should always adhere to [WCAG 2.1 guidelines](https://www.w3.org/TR/WCAG21/).
+In most cases the guidelines provide relatively straightforward things to be aware of, such as:
+
+- **Ensure images are not the sole means to convey information.**
+  Images should only supplement the text.
+  For instance, if you want to include a visual depiction of data such as a graph, then your blog post should adequately explain those data points in the surrounding text.
+- **Images must have suitable alternative text.**
+  The `alt` attribute should always be present for any image and provide a short, meaningful description.
```

Comment:
It may be worth adding a specific .md example here to save blog posters from digging it up. something like - https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#images